docs/tutorial/notebook/reading_a_config_file.ipynb

```diff
@@ -0,0 +1,608 @@
+{
```

Comment:
<p>Mention that items can be viewed as well as updated like in a standard Python dict. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>